### PR TITLE
Update default-settings.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@blowdart  please review

File: aspnetcore/security/data-protection/configuration/default-settings.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Default Settings | Microsoft Docs
+title: Key management and lifetime | Microsoft Docs
```
Comment (tdykstra) [inline aspnetcore/security/data-protection/configuration/default-settings.md]: You changed the title but not the H1

**Comment by Rick-Anderson** [comment]
@tdykstra  arg, fixed

**Comment by blowdart** [comment]
LGTM

**Pull request merged**